Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1903,6 +1903,29 @@ header: <WebCore/ResourceRequest.h>
 };
 #endif
 
+[Nested] struct WebCore::ResourceError::IPCData {
+    [Validator='*type != WebCore::ResourceError::Type::Null'] WebCore::ResourceError::Type type;
+#if PLATFORM(COCOA)
+    RetainPtr<NSError> nsError;
+    bool isSanitized;
```

Comment:
It would probably be nice to just change the type to be consistent across all platforms.